pkgs/applications/networking/mailreaders/nylas-mail/default.nix

```diff
@@ -0,0 +1,136 @@
+{ dpkg, fetchurl, lib, pkgs, stdenv, config
+, alsaLib
+, atk
+, cairo
+, coreutils
+, cups
+, dbus
+, desktop_file_utils
+, expat
+, fontconfig
+, freetype
+, gcc-unwrapped
+, gdk_pixbuf
+, glib
+, gnome2
+, libgcrypt
+, libgnome_keyring
+, libnotify
+, makeWrapper
+, nodejs
+, nspr
+, nss
+, pango
+, python2
+, udev
+, wget
+, xorg
+}:
+
+stdenv.mkDerivation rec {
+  name = "${pkgname}-${version}";
+  pkgname = "nylas-mail";
+  version = "2.0.32";
+  subVersion = "fec7941";
+
+  src =
+    if stdenv.system == "x86_64-linux" then
+      fetchurl {
+        url = "https://edgehill.s3-us-west-2.amazonaws.com/${version}-${subVersion}/linux-deb/x64/NylasMail.deb";
```

Comment:
In risk management terms, it means that we are broadening the number of people who we need to trust and that's a bad thing in most environments. If I say "random person", that's what I meant. Having said that, I am of course happy that you made the effort to provided more than there was before, so it's an improvement. Your effort has probably made it easier to package it from source now, so thanks for your efforts again.